dll/win32/syssetup/lang/it-IT.rc

```diff
@@ -133,20 +133,20 @@ END
 
 IDD_PS2MOUSEPROPERTIES DIALOGEX 0, 0, 252, 218
 STYLE WS_CHILD | WS_VISIBLE | WS_CAPTION
-CAPTION "Advanced Settings"
+CAPTION "Impostazioni avanzate"
 FONT 8, "MS Shell Dlg"
 BEGIN
     RTEXT "&Sample Rate:", -1, 27, 35, 90, 8
-    LTEXT "reports per second", -1, 169, 35, 76, 8
+    LTEXT "report per secondo", -1, 169, 35, 76, 8
     COMBOBOX IDC_PS2MOUSESAMPLERATE, 124, 33, 42, 41, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL
     RTEXT "&Wheel Detection:", -1, 27, 55, 90, 8
     COMBOBOX IDC_PS2MOUSEWHEEL, 124, 53, 95, 46, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL
-    RTEXT "&Input buffer length:", -1, 27, 75, 90, 8
-    LTEXT "packets", -1, 169, 75, 26, 8
+    RTEXT "&Dimensione input buffer:", -1, 27, 75, 90, 8
```

Comment:
"input buffer" is not italian.

Reply:
There is no italian translation. We use english terms to indicate and input buffer on network related arguments.